mlkem/kem.c

```diff
@@ -71,14 +119,18 @@ int crypto_kem_keypair(uint8_t *pk, uint8_t *sk) {
  *                (an already allocated array filled with MLKEM_SYMBYTES random
  *bytes)
  **
- * Returns 0 (success)
+ * Returns 0 on success, and -1 if the public key modulus check fails
```

Comment:
Seeing that this is the top-level API, add reference to the relevant section of the FIPS203 spec?